pkg/agent/bakerapi.go

```diff
@@ -100,7 +100,7 @@ func (agentBaker *agentBakerImpl) GetLatestSigImageConfig(sigConfig datamodel.SI
 	if !distro.IsWindowsDistro() {
 		e := toggles.NewEntityFromEnvironmentInfo(envInfo)
 		imageVersionOverrides := agentBaker.toggles.GetLinuxNodeImageVersion(e)
-		if imageVersion, ok := imageVersionOverrides[string(distro)]; ok {
+		if imageVersion, ok := imageVersionOverrides[string(distro)]; ok && imageVersion != "" {
```

Comment:
we should be able to simplify everywhere to how we're doing it now on line 80

Reply:
Done.